homeassistant/components/telegram_bot/strings.json

```diff
@@ -924,10 +924,6 @@
     "proxy_params_auth_deprecation": {
```

Comment:
We can remove the `proxy_params_auth_deprecation` string since the code for it was removed here.